config/webpack.config.js

```diff
@@ -99,14 +100,32 @@ const commonConfig = ({ publicPath, noHash }) => ({
       },
     ],
   },
-  plugins,
+  plugins: plugins(dev),
   devServer: {
-    writeToDisk: true,
+    allowedHosts: 'all',
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept, Authorization',
+    },
+    historyApiFallback: {
+      index: `${publicPath}index.html`,
+    },
+    https: true,
+    port: 1337,
+    ...proxy({
+      env: 'stage-beta',
+      port: 1337,
+      appUrl: [/^\/*$/, /^\/beta\/*$/],
```

Comment:
![image](https://user-images.githubusercontent.com/50696716/140918827-80300820-a98e-483e-9a1d-31bf67418b67.png)
the `appUrl` makes the terminal output a bit weird, does it accept the regular expressions?

Reply:
Yeah, that is an unfortunate output. I am not sure if there is a better way of representing regexp.